### ISSUE TITLE
elpy.el:3551:7:Error: ‘add-to-list’ can’t use lexical var ‘candidates’

### ISSUE THREAD

**Issue by Curiosidad-Racional** [jorgenschaefer/elpy#1461]
elpy.el:3551:7:Error: ‘add-to-list’ can’t use lexical var ‘candidates’; use
    ‘push’ or ‘cl-pushnew’

# Steps to reproduce

try to byte-compile elpy.py file

# My configuration
## OS
Windows and Linux

## Result of `(elpy-config)`
```
Elpy Configuration

Virtualenv........: None
RPC Python........: 3.6.5 (/usr/bin/python3)
Interactive Python: python3 (/usr/bin/python3)
Emacs.............: 26.1
Elpy..............: 1.24.0
Jedi..............: 0.12.1
Rope..............: 0.10.5
Autopep8..........: 1.3.5 (1.4 available)
Yapf..............: 0.22.0 (0.24.0 available)
Black.............: 18.6b4
Syntax checker....: flake8 (/usr/bin/flake8)
```

## Elpy configuration in my init.el
```

```

**Comment by galaunay** [comment]
Thanks for the report.
It should be fixed by #1464.

**Comment by galaunay** [comment]
Should be fixed now.
Please re-open this issue if it is not the case.
